cellprofiler/gui/figure.py

```diff
@@ -650,10 +650,14 @@ def on_mouse_move(self, evt):
             y0 = evt.ydata
             y1 = evt.ydata
         else:
-            x0 = min(self.mouse_down[0], evt.xdata)
-            x1 = max(self.mouse_down[0], evt.xdata)
-            y0 = min(self.mouse_down[1], evt.ydata)
-            y1 = max(self.mouse_down[1], evt.ydata)
+            x0, x1 = self.mouse_down[0], self.mouse_down[0]
+            y0, y1 = self.mouse_down[1], self.mouse_down[1]
+            if evt.xdata != None:
```

Comment:
I think the proper comparison here might be `if evt.xdata is not None`, same with below
```suggestion
            if evt.xdata is not None:
```

Reply:
Yep, thanks!